airflow-core/src/airflow/utils/__init__.py

```diff
@@ -46,5 +46,11 @@ def __getattr__(name: str):
     "xcom": {
         "XCOM_RETURN_KEY": "airflow.models.xcom.XCOM_RETURN_KEY",
     },
+    "weight_rule": {
+        "WeightRule": "airflow.sdk.definitions.weight_rule.WeightRule",
+        "db_safe_priority": "airflow.sdk.definitions.weight_rule.db_safe_priority",
+        "DB_SAFE_MINIMUM": "airflow.sdk.definitions.weight_rule.DB_SAFE_MINIMUM",
+        "DB_SAFE_MAXIMUM": "airflow.sdk.definitions.weight_rule.DB_SAFE_MAXIMUM",
```

Comment:
These are only used in task sdk and that too in BaseOperator only. Move the constants and the method into BaseOperator i'd say. No point of keeping a module. And also the tests.